lib/cc/analyzer/container.rb

```diff
@@ -27,16 +27,14 @@ def initialize(
         @command = command
         @listener = listener
         @timeout = timeout
-
-        @output_delimeter = "\n"
+        @output_delimiter = "\n"
         @on_output = ->(*) { }
-
         @timed_out = false
         @stderr_io = StringIO.new
       end
 
-      def on_output(delimeter = "\n", &block)
-        @output_delimeter = delimeter
+      def on_output(delimiter = "\n", &block)
+        @output_delimiter = delimiter
```

Comment:
Nice catch, but what do you think about pulling this to its own PR? This logic is so critical and so futzy, I want to be super careful with any changes to it.